src/tools/findImage.ts

```diff
@@ -16,6 +18,8 @@ const findImage = (category: string) => {
     western: categoryWestern,
     asian: categoryAsian,
     etc: categoryEtc,
+    favoriteFilled: favoriteFilled,
+    favoriteLined: favoriteLined,
```

Comment:
key값과 value 값이 동일하다면 아래와 같이 축약하여 작성할 수 있습니다.

```suggestion
    favoriteFilled,
    favoriteLined,
```

Reply:
반영했습니다.